solr/solr-ref-guide/modules/query-guide/pages/learning-to-rank.adoc

```diff
@@ -811,8 +811,8 @@ Related links:
 * {solr-javadocs}/modules/ltr/org/apache/solr/ltr/feature/Feature.html[Feature javadocs]
 * {solr-javadocs}/modules/ltr/org/apache/solr/ltr/norm/Normalizer.html[Normalizer javadocs]
 * {solr-javadocs}/modules/ltr/org/apache/solr/ltr/interleaving/Interleaving.html[Interleaving javadocs]
-* https://cwiki.apache.org/confluence/display/solr/HowToContribute
-* https://cwiki.apache.org/confluence/display/LUCENE/HowToContribute
+* https://github.com/apache/solr/blob/main/CONTRIBUTING.md[Contributing to Solr]
```

Comment:
i wasnt' sure about having these links at all, but in reading through the page, now i see, makes sense.  thanks for fixing.